spec/src/main/doc/javase/scopescontext_se.asciidoc

```diff
@@ -0,0 +1,58 @@
+[[contexts_se]]
+
+== Scopes and contexts in Java SE
+
+[[builtin_contexts_se]]
+
+=== Context management for built-in scopes in Java SE
+
+When running in Java SE, the container must extend the rules defined in <<builtin_contexts>> and is also required to ensure the following rules for built-in context implementation.
+
+
+[[request_context_se]]
+
+==== Request context lifecycle in Java SE
+
+When running in Java SE, the container must extend the rules defined in <<request_context>> and is also required to ensure the following rules.
+
+The request scope is active:
+
+* during any method invocation.
```

Comment:
DeltaSpike for instance. Jozef, what is your suggestion here?
Le mar. 23 juin 2015 à 14:19, Jozef Hartinger <notifications@github.com> a
écrit :

> In spec/src/main/doc/javase/scopescontext_se.asciidoc
> <https://github.com/cdi-spec/cdi/pull/251#discussion_r33034520>:
>
> > +[[builtin_contexts_se]]
> > +
> > +=== Context management for built-in scopes in Java SE
> > +
> > +When running in Java SE, the container must extend the rules defined in <<builtin_contexts>> and is also required to ensure the following rules for built-in context implementation.
> > +
> > +
> > +[[request_context_se]]
> > +
> > +==== Request context lifecycle in Java SE
> > +
> > +When running in Java SE, the container must extend the rules defined in <<request_context>> and is also required to ensure the following rules.
> > +
> > +The request scope is active:
> > +
> > +* during any method invocation.
>
> Then I do not understand why we are adding this in the first place. What
> libraries are not going to work?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cdi-spec/cdi/pull/251/files#r33034520>.
>